index.html

```diff
@@ -0,0 +1,114 @@
+    <link rel="stylesheet" href="style.css">
+    <title>Quick startup</title>
+</head>
+
+<body>
+    <!--This wraps on my page together-->
+    <div class="container">
+        <header class="header-page">
+            <!--header contents goes here-->
+
+            <nav>
+                <!--This is my Logo-->
+                <h3>Solorics</h3>
+
+                <!--Toggle button goes here-->
+                <img src="images/Union.png" alt="Toggle_Image_Button" class="toggle">
+            </nav>
+        </header>
+        <main class="main-page">
+            <!--main contents goes here-->
+            <div>
+                <h3>Opeyemi</h3>
+                <h4>Software Developer</h4>
+                <h5>I can help you build a product , feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</h5>
+
+               <hr>
+
+            </div>
+
+
+        </main>
+
+        <div class="section2">
+
+
+            <div class="heading-left">
+                <h4 class="list-heading">Language</h4>
+                <ul>
+                    <li class="items">Javascript</li>
+                    <li class="items">Ruby</li>
+                    <li class="items">Html</li>
+                    <li class="items">Css</li>
+                </ul>
+            </div>
+
+
+
+
+
+            <div class="heading-right">
+                <h4 class="list-heading">Frameworks</h4>
+                <ul>
+                    <li class="items">Bootstrap</li>
+                    <li class="items">Ruby on Rails</li>
+                    <li class="items">RSPec</li>
+                    <li class="items">CapyBara</li>
+                    <li class="items">Selenium</li>
+                </ul>
+
+            </div>
+
+
+
+            <div class="third-heading">
+                <h4 class="list-heading">Skills</h4>
+                <ul>
+                    <li class="items">Codekit</li>
+                    <li class="items">Github</li>
+                    <li class="items">Codepen</li>
+                    <li class="items">Gitlab</li>
+                    <li class="items">Terminal</li>
+                </ul>
+
+            </div>
+        </div>
+
+        <footer class="footer-page">
+            <!--footer contents goes here-->
+            <hr>
+
+            <div class="icon">
+
+                <img src="images/github1.png" alt="github" class="icons">
+               <div>
+                <img src="images/linkden.png" alt="linkden1" class="icons ">
+                <img src="images/linkden2.png" alt="linkden1" class="icons ">
+               </div>
+                <img src="images/1.png" alt="1" class="icons"><img src="images/twitter1.png" alt="twitter"
+                    class="icons">
+                <img src="images/2.png" alt="2" class="icons">
+            </div>
+        </footer>
```

Comment:
- [ ] The container of the section should be `section` tag. Let's put all of these elements in a `<section>` tag.
we always use section tags in similar situations because it represents a generic standalone section of the document.  Cheers 👍